From 1421911ed3d51cbace6ea5e495b29284ccdb6492 Mon Sep 17 00:00:00 2001
From: Paolo Pisati <paolo.pisati@canonical.com>
Date: Thu, 10 Jan 2013 20:42:27 +0000
Subject: [PATCH 1/2] staging/omapdrm: garbage collect OMAP_DSS_DISPLAY_SUSPENDED
Patch-Mainline: No, solved differently upstream

Compilation fix - leftover from:

commit 998c336d4c7183301ed6a6ca93952f63e3cf694f
Author: Tomi Valkeinen <tomi.valkeinen@ti.com>
Date:   Wed May 30 13:26:00 2012 +0300

    OMAPDSS: remove omap_dss_device's suspend/resume

Cc: stable <stable@vger.kernel.org> # v3.7
Signed-off-by: Paolo Pisati <paolo.pisati@canonical.com>
Signed-off-by: Alexander Graf <agraf@suse.de>
---
 drivers/staging/omapdrm/omap_connector.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/staging/omapdrm/omap_connector.c b/drivers/staging/omapdrm/omap_connector.c
index 91edb3f..ff25467 100644
--- a/drivers/staging/omapdrm/omap_connector.c
+++ b/drivers/staging/omapdrm/omap_connector.c
@@ -113,9 +113,6 @@ static void omap_connector_dpms(struct drm_connector *connector, int mode)
 	if (mode == DRM_MODE_DPMS_ON) {
 		/* store resume info for suspended displays */
 		switch (dssdev->state) {
-		case OMAP_DSS_DISPLAY_SUSPENDED:
-			dssdev->activate_after_resume = true;
-			break;
 		case OMAP_DSS_DISPLAY_DISABLED: {
 			int ret = dssdev->driver->enable(dssdev);
 			if (ret) {
-- 
1.7.12.4

